resources/keb/templates/deployment.yaml

```diff
@@ -201,21 +203,23 @@ spec:
               value: {{ .Values.gardener.kubeconfigPath }}
             - name: APP_KUBECONFIG_ALLOW_ORIGINS
               value: "{{ .Values.kubeconfig.allowOrigins }}"
-            - name: APP_INFRASTRUCTURE_MANAGER_KUBERNETES_VERSION
+            - name: APP_broker_KUBERNETES_VERSION
```

Comment:
APP_BROKER_KUBERNETES_VERSION

Reply:
Side effect of Replace in files. My bad.